Issue/12 UI part to report (#13)
teamulster2/report: report/03_methodology/01_project_management.md

## Project management

Overall, the project took place within five weeks and five participants who worked full time on the project. 
The decisions in the area of project management are described in more detail below.

> 

Reviewer: perhaps add here something like that:

Each team member was assigned a role for which he was responsible. The roles were e.g. 'Git responsible', 'Moderator' or 'Scrum master'.

### Agile project development with scrum

The project management method SCRUM in a simplified version was chosen for the project. 
This had the advantage that, on the one hand, programming was agile under certain rules, on the other hand some times and some overhead were reduced.

The Daily Scrum Meeting was an important part of the daily routine, which was occasionally expanded to talk about the current status and follow up on the relevant issues. 
Due to the limited project time of five weeks, it was decided against the use of sprints and the additional overhead. 

A retrospective meeting was also held once a week, which should contain the current problems and suggestions for improvement, but also positive aspects of the project. 
That helped the project flow and the satisfaction of the participants.

### Open Source Software

The soTired project emerged as a university project in which all participants decided to implement an open source solution. 

> This means that both, so the client application code and the server code are publicly available under the license * insert license here *. 

Reviewer: ```suggestion
This means that both, so the client application code and the server code are publicly available under the license GNU Affero General Public License v3.0. 
```

## Project management

Overall, the project took place within five weeks and five participants who worked full time on the project. 
The decisions in the area of project management are described in more detail below.

### Agile project development with scrum

The project management method SCRUM in a simplified version was chosen for the project. 
This had the advantage that, on one hand, programming was agile under certain rules, on the other hand some times and some overhead was reduced.

The Daily Scrum Meeting was an important part of the daily routine, which was occasionally expanded to talk about the current status and follow up on the relevant issues. 
Due to the limited project time of five weeks, it was decided against the use of sprints and the additional overhead. 

> A retrospective meeting was also held once a week, which should contain the current problems and suggestions for improvement, but also positive aspects of the project. 

Reviewer 2: ```suggestion
A retrospective meeting was also held once a week where current problems regarding the development process could be addressed and improvement suggestions were made. Nevertheless, positive aspects were not neglected either.
```

This complemented the project workflow and the satisfaction of the participants.

> Each team member was assigned a role for which he was responsible. The roles were e.g. 'Moderator', 'Scrum master' or 'Git responsible person'.

Reviewer 2: ```suggestion
Each team member was assigned a role which he was responsible for. The roles were e.g. 'Moderator', 'Scrum master' or 'GitHub repository moderator'.
```

### Open Source Software

The soTired project emerged as a university project in which all participants decided to implement an open source solution. 
This means that both, the client application source code and the server source code are publicly available under the license *GNU Affero General Public License v3.0*. 
Public repositories result in the fact that everyone can view the source code, edit it and use it for their own purposes if it still follows the rules of the original license. 

> This has the advantage that the study participants can see the code and which data is used and stored. This ensures data security. 

Reviewer 2: ```suggestion
This has the advantage that the study participants can see the code and which data is used and stored. Because of this data security is ensured. 
```

> In addition, the code used here is made available to science projects and can be further developed. 

Reviewer 2: ```suggestion
Additionally, all of soTired's source code is made available to science projects and can be further developed.
```

Since the study project is very limited in time, there are some areas of the app that can be added and adapted later on. 

> This can increase the quality of the code in some way.

Reviewer 2: ```suggestion
This may increase code quality.
```
